### ISSUE TITLE
rtags-get-summary-text can't deal with #include "..." if include path is not relative to '.'

### ISSUE THREAD

**Issue by Artem-B** [Andersbakken/rtags#886]
Example:
src/foo.c:
```
#include "bar/foo.h"
void foo() {
  barfoo();
}
```
include/bar/foo.h:
```
// just a function decl
void barfoo();
```
Compile with `cc -I include -c foo.c`

When rtags-get-summary-text is called, it invokes `rc ... --elisp` w/o `-K` and gets back relative include path. Alas, it's relative to one of compiler's include paths. It's **not** relative to the location of foo.c, so when rtags attempts to open `bar/foo.h` using relative file name, it expands to `/top/of/tree/src/bar/foo.h` and fails.

Adding `"-K"` to `rtags-call-rc` in `rtags-symbol-info-internal` appears to fix the problem, but I'm not sure if that's the proper way to deal with the problem.

**Comment by chep** [comment]
Hi,

Same problem here.
And "-K" seems to fix it.

**Comment by Andersbakken** [comment]
I believe this was fixed by fb6fafe7
